portal/src/main/webapp/WEB-INF/jsp/oncoprint/main.jsp

```diff
@@ -96,13 +98,14 @@
                </button>
                <div class="dropdown-menu" style="padding: 10px 5px; width: 270px;min-width: 270px;">
                    <form action="" style="margin-bottom: 0;">
-                       <span>Data type:</span><br>
-                       <input type="radio" name="datatype" style="margin-left:10px" value="sample"> Events per sample</input><br>
-                       <input type="radio" name="datatype" style="margin-left:10px" value="patient"> Events per patient</input><br>
-                       <br>
-                       <input type="checkbox" name="show_unaltered" value="show_unaltered"> Show unaltered columns</input><br>
-                       <input type="checkbox" name="show_whitespace" value="show_whitespace"> Show whitespace between columns</input><br>
-                       <input type="checkbox" name="show_clinical_legends" value="show_clinical_legends"> Show legends for clinical tracks</input>
+                       <fieldset style="border: 0; padding-bottom: 0;">
```

Comment:
There is no HTTP action for this form, the selection is interpreted by JavaScript:
```JavaScript
	    // initialize radio buttons
	    $(toolbar_selector).find('#oncoprint_diagram_view_menu')
				   .find('input[type="radio"][name="datatype"][value="'+
				   (State.using_sample_data ? "sample" : "patient") + '"]')
				   .prop("checked", true);
	    
	    $(toolbar_selector).find('#oncoprint_diagram_view_menu')
				   .find('input[type="radio"][name="datatype"]').change(function(e) {
		State.using_sample_data = $(toolbar_selector).find('#oncoprint_diagram_view_menu')
				   .find('input[type="radio"][name="datatype"]:checked').val() === 'sample';
		if (State.using_sample_data) {
		    State.setDataType('sample');
		} else {
		    State.setDataType('patient');
		}
		updateHeaderBtnText();
		updateDownloadIdOrderText();
	    });
```